src/widget/wlibrarytableview.h

```diff
@@ -42,6 +55,8 @@ class WLibraryTableView : public QTableView, public virtual LibraryView {
     void loadVScrollBarPosState();
     void saveVScrollBarPosState();
 
+    QMap<QString, int> m_vScrollBarPosValues;
```

Comment:
I'm quite new to C++, so I wasn't sure about that. Initially I wanted to in that way. I can still change it if needed.